modules/mod_ginger_collection/filters/filter_collection_query_pager.erl

```diff
@@ -0,0 +1,29 @@
+%% @doc A date filter for ISO 8601 datetimes (instead of datetime tuples).
+%%      If a year is passed <<"2017">>, that year is printed. Other inputs are
+%%      converted to a datetime tuple set and passed to Zotonic's date filter.
+-module(filter_collection_query_pager).
+
+-export([
+    collection_query_pager/5
+]).
+
+-include_lib("zotonic.hrl").
+
+collection_query_pager(Search, Index, QueryId, "1", Context) ->
+    Search2 = z_convert:to_atom(Search),
+    Result = z_search:search({Search2, [{query_id, QueryId},{index, Index}]},{1,2},Context),
+    Ids = Result#search_result.result,
+    Prev = undefined,
+    Next = lists:last(Ids),
+    [Prev, Next];
```

Comment:
Writing this in fewer lines of code is clearer to me:

```erlang
#search_result{result = Ids} = z_search:search({Search2, [{query_id, QueryId}, {index, Index}]}, {1, 2}, Context),
[undefined, lists:last(Ids)].
```

Reply:
Fixed